### DISCUSSION TITLE
m1运行求助

### DISCUSSION THREAD

**Discussion by SuchanTso** [BoomingTech/Piccolo#393] [Q&A]
照着做了操作编译成功，但是一直崩溃
<img width="1512" alt="image" src="https://user-images.githubusercontent.com/56480275/203305374-2eaf6071-5067-4c27-abd6-b9cf57fd0463.png">

**Comment by ShenMian** [comment]
看起来像 JoltPhysics 的问题, 但看 JPH 的 [README](https://github.com/jrouwe/JoltPhysics#supported-platforms) 应该是支持 M1 的.

**Reply by SuchanTso** [reply]
大佬能指导下怎么弄嘛
